source/_integrations/home_connect.markdown

```diff
@@ -94,6 +94,7 @@ The integration configuration will ask for the *Client ID* and *Client Secret* c
 {% note %}
 
 - The entities availability depends on the appliance type, but the appliance might not support all the entities for its type.
+- The program option entities will not be provided by the integration until they are available in the actual program.
```

Comment:
```suggestion
- The program option entities will not be provided by the integration until they are available in the selected program.
```

Reply:
Yeah, I has this doubt also.

You can have the options of the selected programs while the program is not running, but once it has started, the options you have available are the ones from the active program, I didn't know how to reflect that in the documentation without extending too much, but maybe I should extend that much to be clear about that

Reply:
Maybe this is enough?
```suggestion
- The program option entities will not be provided by the integration until they are available in the selected or active program.
```

Reply:
Seems good